scripts/docker-entrypoint.sh

```diff
@@ -55,10 +55,15 @@ migrations() {
     fi
 }
 
+collectstatic() {
+    sudo -HEu paperless "/usr/src/paperless/src/manage.py" "collectstatic" "--clear" "--no-input"
+}
+
 initialize() {
     map_uidgid
     set_permissions
     migrations
+    collectstatic
```

Comment:
Rather than collecting the static files on every startup, since they depend on the code-base that doesn't change dynamically, I'd suggest collecting them within the Dockerfile. This offloads the work from the user to the builder, while only adding ~10MB in image size (485MB before vs. 495MB after), which I think is worth it.

This is my suggested (and tested) change:

```diff
diff --git a/Dockerfile b/Dockerfile
index cfcfbd2..d5400aa 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -65,3 +65,6 @@ COPY src/ /usr/src/paperless/src/
 COPY data/ /usr/src/paperless/data/
 COPY media/ /usr/src/paperless/media/

+# Collect static files
+RUN sudo -HEu paperless /usr/src/paperless/src/manage.py collectstatic --clear --no-input
+
diff --git a/scripts/docker-entrypoint.sh b/scripts/docker-entrypoint.sh
index 303d1b8..5cfcd40 100644
--- a/scripts/docker-entrypoint.sh
+++ b/scripts/docker-entrypoint.sh
@@ -55,15 +55,10 @@ migrations() {
     fi
 }

-collectstatic() {
-    sudo -HEu paperless "/usr/src/paperless/src/manage.py" "collectstatic" "--clear" "--no-input"
-}
-
 initialize() {
     map_uidgid
     set_permissions
     migrations
-    collectstatic
 }

 install_languages() {
```

Reply:
That's a great idea - thank you! I'll give it a whirl on my setup and integrate your suggested changes asap.